perfectscale/components/external-secret.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: <CLUSTER_NAME>-perfectscale-secret
+  namespace: perfectscale
+spec:
+  target:
+    name: perfectscale-secret
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: vault-kv-secret
+  refreshInterval: "10s"
+  dataFrom:
+    - extract:
+        key: /perfectscale
```

Comment:
I imagine this is going to be an achilles heel... Is this from their tokens or something along those lines?